src/cpp/transport/UDPTransportInterface.cpp

```diff
@@ -638,9 +637,9 @@ LocatorList_t UDPTransportInterface::ShrinkLocatorLists(const std::vector<Locato
                                 {
                                     // Custom Loopback locator
                                     pendingUnicast.push_back(*it);
-                                    break;
                                 }
                             }
+                            break;
```

Comment:
This `break` in the old code wasn't present in case both `loopbackLocator`, and `*it` aren't allowed, but in the new version, in that case, it will stop comparing the locator `*it` against the local interfaces. Is this the desired behavior now?